### PR TITLE
 added SVG  image preview format support

### PR DESCRIPTION
**Pull request by Thanush19**
#### Checklist

 - [ ] I have updated tests where applicable.
 - [x] I have added an entry to the CHANGELOG.

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -49,6 +49,6 @@
         "react-test-renderer": "^17.0.2",
         "redux-mock-store": "^1.5.4",
         "through2": "^4.0.2",
-        "typescript": "^4.3.5"
+        "typescript": "^5.3.3"
```
Comment (mhils) [inline web/package.json:52]: Please submit dependency bumps as a separate PR.

File: web/src/js/components/contentviews/HttpMessage.tsx [hunk 2]
```diff
@@ -141,15 +141,18 @@ type ViewImageProps = {
...
-                src={MessageUtils.getContentURL(flow, message)}
-                alt="preview"
-                className="img-thumbnail"
-            />
+            <object
+                data={imageContentURL}
+                type="image/*"
+                width="100%"
+                height="100%"
+            ></object>
```
Comment (mhils) [inline web/src/js/components/contentviews/HttpMessage.tsx:155]: I don't understand this here - why does the img tag not work?

Comment (Thanush19) [reply web/src/js/components/contentviews/HttpMessage.tsx:155]: @mhils  kindly  review PR #6551  ,
